docs/user/dashboard/dashboard.asciidoc

```diff
@@ -424,41 +424,45 @@ Ready to try out Timelion? For step-by-step tutorials, refer to:
 [[timelion-deprecation]]
 ==== Timelion app deprecation
 
-Deprecated since 7.0, the Timelion app will be removed in 8.0. If you have any Timelion worksheets, you must migrate them to a dashboard.
```

Comment:
Not "the Timelion app"?

Reply:
That was my initial thought too, but the screenshot in https://github.com/elastic/kibana/pull/74660 shows Timelion app instead of the Timelion app, so I used the same.